projects/vdk-core/src/vdk/internal/builtin_plugins/connection/managed_cursor.py

```diff
@@ -126,15 +132,34 @@ def execute(
                 raise e
 
     def _decorate_operation(self, managed_operation: ManagedOperation, operation: str):
-        if self.__connection_hook_spec.db_connection_decorate_operation.get_hookimpls():
+        decorate_operation = None
+
+        if (
+            self.__managed_database_connection
```

Comment:
Can you please add comments explaining that this checks if the IDatabaseManagedConnection method has been overridden?

Reply:
done